hack/build.sh

```diff
@@ -25,23 +25,21 @@ trap "rm -f ${DOCKERFILE}.${version}" SIGINT SIGQUIT EXIT
 function docker_build_with_version {
   cp ${DOCKERFILE} "${DOCKERFILE}.${version}"
   git_version=$(git rev-parse HEAD)
-  sed -i "${DOCKERFILE}.${version}" -e "s/NODE_VERSION *= *.*/NODE_VERSION=${version} \\\/"
+  sed -i.bak -e "s/NODE_VERSION *= *.*/NODE_VERSION=${version} \\\/" "${DOCKERFILE}.${version}"
   echo "LABEL io.origin.builder-version=\"${git_version}\"" >> "${DOCKERFILE}.${version}"
   docker build -t ${IMAGE_NAME}:${version} -f "${DOCKERFILE}.${version}" .
   if [[ "${SKIP_SQUASH}" != "1" ]]; then
     squash "${DOCKERFILE}.${version}"
   fi
-  rm -f "${DOCKERFILE}.${version}"
+  rm -f "${DOCKERFILE}.${version}" "${DOCKERFILE}.${version}.bak"
 }
 
 # Install the docker squashing tool[1] and squash the result image
 # [1] https://github.com/goldmann/docker-squash
 function squash {
-  # FIXME: We have to use the exact versions here to avoid Docker client
-  #        compatibility issues
-  easy_install -q --user docker_py==1.6.0 docker-squash==1.0.0rc6
+  pip install -q --user -U docker-squash
   base=$(awk '/^FROM/{print $2}' $1)
-  ${HOME}/.local/bin/docker-squash -f $base ${IMAGE_NAME}:${version}
+  $(python -m site --user-base)/bin/docker-squash -f $base ${IMAGE_NAME}:${version}
```

Comment:
you're welcome to just delete the squashing logic entirely, we're moving away from it.